### ISSUE TITLE
bug in dataframe_from_urls

### ISSUE THREAD

**Issue by ritviksahajpal** [benmack/nasa_hls#3]
thanks for a great package! I have a small bug-fix to suggest. Currently the code fails when there is a tile with a 'T' in the name e.g. 35TPL because of the replace statement in the function below:

```
def dataframe_from_urls(urls):
    """Convert list of HLS dataset URLs in dataframe (columns: product, tile, date, url)."""
    import pandas as pd
    datasets = pd.DataFrame(pd.Series(urls).str.split("/", expand=True))[11] \
        .str.split(".", expand=True)[[1, 2, 3]] \
        .rename({1: "product", 2: "tile", 3: "date"}, axis=1)
    datasets.tile = datasets.tile.str.replace("T", "")
    datasets.date = pd.to_datetime(datasets.date, format="%Y%j")
    datasets["url"] = urls
    return datasets
```

Instead, if you put a `n=1` in the replace statement, it will only replace the first `T` in `T35TPL` and the bug will be fixed:

```
def dataframe_from_urls(urls):
    """Convert list of HLS dataset URLs in dataframe (columns: product, tile, date, url)."""
    import pandas as pd
    datasets = pd.DataFrame(pd.Series(urls).str.split("/", expand=True))[11] \
        .str.split(".", expand=True)[[1, 2, 3]] \
        .rename({1: "product", 2: "tile", 3: "date"}, axis=1)
    datasets.tile = datasets.tile.str.replace("T", "",n=1)
    datasets.date = pd.to_datetime(datasets.date, format="%Y%j")
    datasets["url"] = urls
    return datasets
```

**Comment by benmack** [comment]
Thanks a lot for the report and the fix. I am happy that the package is useful for you! 
I am on vacation and fix it as soon as I am back next week.

**Comment by benmack** [comment]
looks like I already had a a fix for the issue in a uncommitted local change.
It should be solved with b616d1a517cce81f55fbb317469821872efd9324 in the develop branch and with be73e5abd0ed73fbd9ec85f58c2a10cafff28918 in the master branch. 

Feel free to re-open in case necessary.

**Comment by ritviksahajpal** [comment]
thanks!
